darkreader.css

```diff
@@ -27671,4 +27674,9 @@ div.c-scrollbar__child > div.c-virtual_list__scroll_container,
 .p-invite_users_input__input_subtext,
 .p-channel_create_modal__private_toggle_copy_body {
     color: #bdb9b2;
-}
\ No newline at end of file
+}
+/* Windows user popup */
+.p-member_profile_card, .p-member_profile_card * {
+    background-color: #17181c !important;
+    color: #d6d4cf !important;
+}
```

Comment:
The user's popup was white, so changing the background color and font color of it (and everything inside of it) swapped it to dark and made everything readable.